benchmarks/README.rst

```diff
@@ -22,9 +22,6 @@ By default, `asv` ships with support for anaconda and virtualenv::
     pip install asv
     pip install virtualenv
 
-After contributing new benchmarks, you should test them locally
-before submitting a pull request.
-
```

Comment:
Is there a reason to remove this?

Reply:
Sorry, I was trying a few changes of phrase. I think I'm happier with this line (and explaining in the next section that there's not much point running the benchmark replications.

IMO its a little confusing, because the benchmarks take a long time, we should possibly default to storing the results (in a `gitignore`'d folder perhaps) but that's a different discussion.